server/jest.config.js

```diff
@@ -3,4 +3,7 @@ module.exports = {
   preset: "ts-jest",
   testEnvironment: "node",
   setupFiles: ["<rootDir>/test/setup-tests.js"],
+  transform: {
+    "^.+\\.(ts|tsx)$": "ts-jest",
```

Comment:
What is it good for?

Reply:
damit jest .ts files mit ts-jest transpiled